gatsby-node.js

```diff
@@ -82,7 +82,9 @@ async function sourcePrimerReactData({actions, createNodeId, createContentDigest
   // Save the current version of Primer React to the GraphQL store.
```

Comment:
This doesn't seem relevant to this PR. I'm fine with these updates, but we should move them to a separate PR if they continue to block CI checks from passing.

Reply:
Oh this was a fix that I merged from master but wasn't merged yet. This is gone now.